src/Ui.Playground/Ui.Playground.csproj

```diff
@@ -1,28 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-    <AssemblyName>GraphQL.Server.Ui.Playground</AssemblyName>
-    <RootNamespace>GraphQL.Server.Ui.Playground</RootNamespace>
-    <Product>GraphQL for .NET</Product>
-    <Company />
-    <Authors>Pekka Heikura</Authors>
-    <Description>GraphQL playground extension</Description>
-    <PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
-    <RepositoryType>Git</RepositoryType>
-    <PackageTags>GraphQL Playground</PackageTags>
-    <Copyright>Pekka Heikura</Copyright>
-    <PackageId>GraphQL.Server.Ui.Playground</PackageId>
-  </PropertyGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Html\index.html" />
-  </ItemGroup>
+	<PropertyGroup>
+		<TargetFramework>netcoreapp2.0</TargetFramework>
+		<AssemblyName>GraphQL.Server.Ui.Playground</AssemblyName>
+		<RootNamespace>GraphQL.Server.Ui.Playground</RootNamespace>
+		<Product>GraphQL for .NET</Product>
+		<Company />
+		<Authors>Pekka Heikura</Authors>
+		<Description>GraphQL playground extension</Description>
+		<PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
+		<RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
+		<RepositoryType>Git</RepositoryType>
+		<PackageTags>GraphQL Playground</PackageTags>
+		<Copyright>Pekka Heikura</Copyright>
+		<PackageId>GraphQL.Server.Ui.Playground</PackageId>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.1" />
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="2.0.0" />
-  </ItemGroup>
+
+	<ItemGroup>
+		<EmbeddedResource Include="Internal\playground.cshtml" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.1" />
+		<PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.1" />
+		<PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="2.0.0" />
```

Comment:
Some of these can now be removed as they're not required anymore.

Reply:
Abstraction should stay. Its where HttpContext life